### ISSUE TITLE
Dependancy failure

### ISSUE THREAD

**Issue by docbill** [odeke-em/drive#803]

**Comment by odeke-em** [comment]
Hello there @docbill, is there a way you can do a `go get -u golang.org/x/sys/unix`?

**Comment by docbill** [comment]
Yes ultimately that is the workaround I had to add to the Dockerfile.   After that I ran into the same problem, with a second dependency.   It is not a huge issue for a Dockerfile since it will reproduce the same build steps every single time.   Just a bit of pain to google the solution... But it definitely makes it harder for anyone trying to build this manually.

**Comment by odeke-em** [comment]
@docbill, gotcha. So wouldn't it be better to instead of depending on godeps(since I didn't need it for mine locally), use the steps in your Dockerfile like this
```
RUN go get -u -v ./cmd/drive && go build -o bin/drive ./cmd/drive
```

**Comment by docbill** [comment]
I added the godeps as the result of a suggestion in an earlier issue reported.   At the time it didn't work without out it.   I haven't tried reverting back to this simpler version of the command recently.

**Comment by odeke-em** [comment]
Ping!
